elements/pl-truth-table/pl-truth-table.py

```diff
@@ -0,0 +1,567 @@
+import math
```

Comment:
Not sure if this is being actively worked on still, but for any work going forward, I __strongly__ encourage the use of type hints. It makes your code much easier to read and verify, and also encourages writing cleaner code.